lib/fhir/core.js

```diff
@@ -236,28 +236,49 @@ module.exports = (mongo, modules) => {
                       return callback(err)
                     }
 
-                    const response = fhirCommon.bundleResults('searchset', results, total, ctx.url)
-                    if (_getpagesoffset + _count < total) {
-                      fhirCommon.addBundleLinkNext(response, ctx.url, _getpagesoffset + _count, _count)
-                    }
-
-                    hooks.executeAfterHooks('search', fhirModule, ctx, resourceType, response, (err, badRequest, results) => {
-                      if (err || badRequest) {
-                        return handleErrorAndBadRequest(err, badRequest, callback)
+                    fhirCommon.mapSearchNameToPath(ctx.query._include, (err, data) => {
+                      if (err) {
+                        logger.info(err)
                       }
 
-                      // format the updated entry array for the response
-                      if (results && results.entry) {
-                        results.entry.forEach((resource) => {
-                          fhirCommon.formatResource(resource.resource)
+                      fhirCommon.includeResources(data, results)
+                        .then((res) => {
+                          if (res && Array.isArray(res)) {
```

Comment:
What happens if it's not an array?

Reply:
@andrew-w-ross  should always be an array or undefined, so the check was just to be safe. Truth be told unnecessary

Reply:
Throw an error if it's not an array